src/views/RecIM/components/Forms/InviteParticipantForm/index.jsx

```diff
@@ -4,17 +4,22 @@ import GordonDialogBox from 'components/GordonDialogBox';
 import { ParticipantList } from './../../List';
 import GordonQuickSearch from 'components/Header/components/QuickSearch';
 import GordonSnackbar from 'components/Snackbar';
-import { addParticipantToTeam } from 'services/recim/team';
+import { addParticipantToTeam, createTeam, deleteTeamParticipant } from 'services/recim/team';
 import { useCallback } from 'react';
 import GordonLoader from 'components/Loader';
 import styles from './InviteParticipantForm.module.css';
+import userService from 'services/user';
+import { useUser } from 'hooks';
 
 const InviteParticipantForm = ({
   closeWithSnackbar,
   openInviteParticipantForm,
   setOpenInviteParticipantForm,
   teamID,
+  soloTeam,
```

Comment:
```suggestion
  individualSport,
```

Can we make this name change? I don't really like soloTeam, it sounds odd.